chatgse/public/prompts.json

```diff
@@ -1,124 +1,29 @@
 {
```

Comment:
Maybe revert this as well; I think the prompts were kind of interesting. Maybe we can add some biomed prompts at the very top, and put this general introduction also on the new "home page". The home page can be in another PR if it does not take priority now.

Reply:
Reverted!